packages/app-store/_utils/stripe.ts

```diff
@@ -24,14 +24,12 @@ export async function getStripeCustomerIdFromUserId(userId: number) {
   return customerId;
 }
 
-const userType = Prisma.validator<Prisma.UserArgs>()({
```

Comment:
I'm curious. What trouble is the select validator causing @joeauyeung ?

Reply:
This was throwing a ton of `prisma.client.validator` is undefined errors in the tests. I think this change is good as well because Prisma recommends using typescript's `satisfies` to reduce overhead.

https://www.prisma.io/blog/satisfies-operator-ur8ys8ccq7zb